### ISSUE TITLE
Support creation of other family members in Enketo

### ISSUE THREAD

**Issue by sglangevin** [medic/medic-webapp#2313]
We currently support the child-repeat section in our Enketo implementation, which, when registering a family, allows a CHP to register multiple children (or other family members) when creating a family. Ideally, we would have more flexibility in creating people during family registration so that we can handle other use cases where CHPs are registering families with different structures. 

For example, another partner wants the ability to do child registration in one section and registration of women who are of childbearing age in another section of the same form. Or we might want to explicitly register a male head of household during family registration. Or we could have a need to create family units within a household that are somewhat independent of one another - this is another way we could be registering polygamous families, where a man is the head of household and each woman represents a sub-unit.

There are probably other examples, but we should look at making family member creation more flexible in order to support variation in family registration workflows.

**Comment by nice-snek** [comment]
hi frand @mandric, U is coot

Please close or schedule before the end of this sprint. See [triaging old issues](https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#triaging-old-issues).

Nice Snek

**Comment by alxndrsn** [comment]
@sglangevin is there an example form available that demonstrates this bug?

**Comment by sglangevin** [comment]
Converted to txt so I could upload it, but you can convert back to xml.
[clinic.txt](https://github.com/medic/medic-webapp/files/863214/clinic.txt)

**Comment by alxndrsn** [comment]
This may be fixed by #2912

**Comment by alxndrsn** [comment]
@sglangevin I'd be grareful if you can check if recent changes have fulfilled the requirement in this ticket.

**Comment by sglangevin** [comment]
Once that other issue is backported to 2.13.x (#3840) then I can also test this one.

**Comment by sglangevin** [comment]
This is still not working for me. I tried with the original form and with using some of the new configuration from #2912 and no luck. Seems like #2912 only works for reports and not for contact forms. This is also not a priority right now as we have found a workaround for registering different types of people from within the same group. I'm going to close this and we can come back to it if there is a specific project that needs it.
